pyramid_oereb/contrib/models/oereblex/theme.py

```diff
@@ -0,0 +1,273 @@
+        pk_type (sqlalchemy.sql.type_api.TypeEngine): The type of the primary column. E.g.
+            sqlalchemy.String or sqlalchemy.Integer or another one fitting the underlying DB
+            needs
+        geometry_type (str): The geoalchemy geometry type defined as well known string.
+        srid (int): The SRID defining the projection of the geometries stored in standard db schema.
+    """
+    Base = declarative_base()
+
+    class Availability(Base):
+        """
+        A simple bucket for achieving a switch per municipality. Here you can configure via the
+        imported data if a public law restriction is available or not. You need to fill it with
+        the data you provided in the app schemas municipality table (fosnr).
+
+        Attributes:
+            fosnr (int): The identifier of the municipality in your system (id_bfs = fosnr)
+            available (bool): The switch field to configure if this plr is available for the
+                municipality or not.  This field has direct influence on the applications
+                behaviour. See documentation for more info.
+        """
+        __table_args__ = {'schema': schema_name}
+        __tablename__ = 'availability'
+        fosnr = Column(pk_type, primary_key=True, autoincrement=False)
+        available = Column(Boolean, nullable=False, default=False)
+
+    class Office(Base):
+        """
+        The bucket to fill in all the offices you need to reference from public law restriction, document,
+        geometry.
+
+        Attributes:
+            id (str): The identifier. This is used in the database only and must not be set manually. If
+                you  don't like it - don't care about.
+            name (dict): The multilingual name of the office.
+            office_at_web (dict): A web accessible url to a presentation of this office (multilingual).
+            uid (str): The uid of this office from https
+            line1 (str): The first address line for this office.
+            line2 (str): The second address line for this office.
+            street (str): The streets name of the offices address.
+            number (str): The number on street.
+            postal_code (int): The ZIP-code.
+            city (str): The name of the city.
+        """
+        __table_args__ = {'schema': schema_name}
+        __tablename__ = 'office'
+        id = Column(pk_type, primary_key=True, autoincrement=False)
+        name = Column(JSONType, nullable=False)
+        office_at_web = Column(JSONType, nullable=True)
+        uid = Column(String(12), nullable=True)
+        line1 = Column(String, nullable=True)
+        line2 = Column(String, nullable=True)
+        street = Column(String, nullable=True)
+        number = Column(String, nullable=True)
+        postal_code = Column(Integer, nullable=True)
+        city = Column(String, nullable=True)
+
+    class DataIntegration(Base):
```

Comment:
Perhaps some more classes in https://github.com/openoereb/pyramid_oereb/blob/master/pyramid_oereb/standard/models/theme.py could be packed into functions and used here as much code is equal in both files.

Reply:
Maybe analogue to the Interlis-Modell, the classes "Document" and "Office" in one function (as it is already; the two classes are used in the main.py) and the other classes in another function.

Reply:
I know, it might be overkill. But instead of grouping them, maybe we offer them each in a function? So any possible future reuse will be covered already.